README.md

```diff
@@ -52,7 +52,6 @@ have enabled by default.
 To disable a rule, set its value to `null`.
```

Comment:
Yeah either is cool - the option was added since some people were confused by you couldn't out false